src/sympc/session/session.py

```diff
@@ -278,7 +278,7 @@ def prrs_generate_random_share(
             share = ReplicatedSharedTensor(
                 shares=[share1, share2],
                 session_uuid=self.uuid,
-                config=Config(encoder_precision=0),
+                # config=Config(encoder_precision=0),
```

Comment:
Q: Remove this?

Reply:
George I have reverted to use no encoding.